### PR TITLE
v2.3.0rc1

### PR DESCRIPTION
**Pull request by forsyth2**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+{% set version = "2.3.0rc1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/E3SM-Project/zppy/archive/v{{ version }}.tar.gz
-  sha256: a233512d612bef036f022b40c9b469eb6d28f6cfcf8f16ebf6dcae946df49d0a
+  sha256: 26c6416ba4d3e184ddc42fa7d6569f5440a6bf53
```
Comment (forsyth2) [inline recipe/meta.yaml:10]: @xylar, I made this the SHA of the last commit to be included. Is that correct?

Comment (forsyth2) [reply recipe/meta.yaml:10]: Actually, the latest commit would be `7a196b55b212fc2d294e332c1f469a5a26c9be72`.

**Comment by xylar** [comment]
> This will upload to the special label we set up: conda-forge e3sm_dev

We need to fix that because I'm having trouble when every dev package goes to the same label.  It seemed like it would be convenient but in fact it means I can't choose at the individual package level if I use a dev or a release package.

**Comment by xylar** [comment]
@mahf708, it looks like you updated the `dev` branch directly without making a PR from a branch from your own fork.This is really not a good idea.  It resulted in new packages being built an uploaded to the `e3sm_dev` label:
https://anaconda.org/conda-forge/zppy/files
Please always use a personal fork and always use PRs to modify any branches on conda-forge to avoid this happening.

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
> @mahf708, it looks like you updated the dev branch directly without making a PR from a branch from your own fork.This is really not a good idea.

As I think you noted, the alternative is to make sure CI gets skipped like you did here:
https://github.com/conda-forge/zstash-feedstock/commit/f188b62508c2d0611d20fc8292b18548242c8989
However, for zstash as for zppy, we want its own `zstash_dev` label, not `e3sm_dev`.

**Comment by xylar** [comment]
@mahf708, I totally agree with all of that.  No harm done.  And it does seem wise to avoid accidentally merging something to a `dev` branch that's missing the label.  So I appreciate what you did for both zstash and zppy to try to prevent that.  Thanks also for changing to the `zstash_dev` label.  That will be helpful.

I do completely appreciate how proactive you've been on all things conda-forge.  It helps a lot to not feel like the only expert on those things!

**Comment by forsyth2** [comment]
@xylar @mahf708 Thank you for finalizing this pull request!

**Pull request merged**